### ISSUE TITLE
No config gui

### ISSUE THREAD

**Issue by hanetzer** [HalogenMods/Chlorine#2]
Pending pr on forge to reimplement their old config menu for mods. Until then,
you can modify `~/.minecraft/config/chlorine-client.toml`

If you change advanced.chunk_renderer_backend, you should press `F3+A` to
cause the changes to propogate.

**Comment by Edulml** [comment]
There is still a bit more you can do to improve user friendliness. Multidraw is the setting that provides most of the performance boost, yet it isn't enabled by default. I thought that you didn't implement it yet until I went here (and also found out why it isn't the default). Still, adding messages about this issue on the main mod page could be a good idea.

**Comment by hanetzer** [comment]
Yes, multidraw is best for hardware that can do it, agreed. However, the check occurs too early in
forge's startup and will cause a crash, and that would be far less friendly than having to manually
set multidraw.

**Comment by Corvus20** [comment]
Being how that forge hasn't really re-implemented their config menu, you might have to consider alternatives, like cloth config.
